pkg/spoke/managedcluster/creating_controller.go

```diff
@@ -53,14 +54,31 @@ func NewManagedClusterCreatingController(
 }
 
 func (c *managedClusterCreatingController) sync(ctx context.Context, syncCtx factory.SyncContext) error {
-	_, err := c.hubClusterClient.ClusterV1().ManagedClusters().Get(ctx, c.clusterName, metav1.GetOptions{})
+	var managedClusterClientConfigs []clusterv1.ClientConfig
+	if len(c.spokeExternalServerURLs) != 0 {
+		for _, serverURL := range c.spokeExternalServerURLs {
+			managedClusterClientConfigs = append(managedClusterClientConfigs, clusterv1.ClientConfig{
+				URL:      serverURL,
```

Comment:
we need to consider the case that user update this, we should not update this back from agent

Reply:
> we need to consider the case that user update this, we should not update this back from agent

@qiujian16 ok, I have change the condition for updating this in https://github.com/open-cluster-management-io/registration/pull/270/files#diff-0b36934623fbbd233f2529b6c4e12a550c7a7537f8d1223070b2c2748140cdb9R74
Plz review again, thx.